pyspark/bigdl/models/rnn/README.md

```diff
@@ -0,0 +1,108 @@
+# Recurrent Neural Network
+
+Model that supports sequence to sequence processing
+
+This is an implementation of Simple Recurrent Neural Networks for Language Modeling. Please refer to the [original paper](http://www.fit.vutbr.cz/research/groups/speech/publi/2010/mikolov_interspeech2010_IS100722.pdf) by Tomas Mikolov.
+
+The implementation of RNNs in this code is referred to in the [Keras Recurrent](https://keras.io/layers/recurrent/) documentation.
+
+
+## Get the BigDL files
+
+Please build BigDL referring to [Build Page](https://github.com/intel-analytics/BigDL/wiki/Build-Page).
+
+
+## Prepare the Input Data
+You can download the Tiny Shakespeare Texts corpus from [here](https://raw.githubusercontent.com/karpathy/char-rnn/master/data/tinyshakespeare/input.txt).
```

Comment:
Would you like to wrap the data loading same as `$BigDL/pyspark/bigdl/dataset/mnist.py` ?